### DISCUSSION TITLE
Run pythoncode without returning a plot

### DISCUSSION THREAD

**Discussion by dflvunoooooo** [leo-colisson/robust-externalize#32] [Q&A] [answered]
Is it possible to specify a preset, which does not expect a plot of some kind? I would like to use python to calculate values returned from previous plots and return those results to latex.

**Comment by dflvunoooooo** [comment]
As a workaround one could make the plot very, very small, with 
```
    plt.rcParams.update({  
        "figure.figsize": "0.0001, 0.0001", 
    })
```

I tryed to use the package pythontex for this, but it is horrible complicated to use. robust-externalize is far easier :)

**Comment by tobiasBora** [comment, marked as answer]
Ahah please don't use your dirty small figure ^^ Sure, just use `do not include pdf` like in:
```
\documentclass{article}

\usepackage{robust-externalize}

\begin{document}

\begin{CacheMeCode}{python, do not include pdf}
  import math
  myvalue = math.sqrt(37)
  write_to_out(r"\gdef\myvalue{" + str(myvalue) + r"}")
\end{CacheMeCode}

The value we computed via the above python code is \myvalue.

\end{document}
```

**Reply by dflvunoooooo** [reply]
Yeah, it was a dirty workaround :) Thank you!
